src/utils/abi-decoder.ts

```diff
@@ -217,12 +221,15 @@ export default class ABIDecoder {
                 let abiItem: ABI_Event | null = this.eventSigs[address] ? this.eventSigs[address][eventID] : null
 
                 let abiItemOptions: ABI_Event[] = []
-
                 // if not, and it's the ambiguous Transfer event, check the contract type
                 if (!abiItem && eventID === transferHash) {
                     abiItem = getABIForTransferEvent(contractDataMap[address].type)
                 }
 
+                if (eventID === transferSingleHashErc1155) {
+                    abiItem = getABIForTransferEvent(contractDataMap[address].type)
+                }
```

Comment:
For a reason we don't know yet, a lot of 1155 contracts still show up as `other` to us, so I think we shouldn't even check if the contract type is 1155, if it's this event hash, we should just override whatever abi they are using and use our default one